powerplantmatching/data.py

```diff
@@ -2142,6 +2143,127 @@ def GEM(raw=False, update=False, config=None):
     data = [f(raw=raw, update=update, config=config) for f in GEMS_FUNTIONS]
     return pd.concat(data, ignore_index=True)
 
+def OSM(raw=False, update=False, config=None):
+    """
+    Importer for the OpenStreetMap power plant data.
+    
+    Parameters
+    ----------
+    raw : boolean, default False
+        Whether to return the original dataset
+    update: bool, default False
+        Whether to update the data from the url.
+    config : dict, default None
+        Add custom specific configuration,
+        e.g. powerplantmatching.config.get_config(target_countries='Italy'),
+        defaults to powerplantmatching.config.get_config()
```

Comment:
```suggestion
        Add overwrites to configuration, e.g. `config={target_countries: 'Italy'}`.
```
See below, this would need to be changed in all docstrings